app/views/tpi/search/index.html.erb

```diff
@@ -21,7 +21,8 @@
       <h3>Companies</h3>
       <ul>
         <% @companies.each do |company| %>
-          <li><%= link_to company.name, tpi_company_path(company) %></li>
+          <% company_name = company.is_active ? company.name : "#{company.name} (Not active)" %>
+          <li><%= link_to company_name, tpi_company_path(company) %></li>
```

Comment:
I’m not sure on which pages to add the inscription "not active" to the name of the companies. 
Perhaps it is necessary to create the “name” method in the “company” model so that the inscription appears on all pages?

Reply:
:+1: maybe move it to view helpers (or decorators for admin pages)?

Reply:
We may need decorators/presenters layer for public interface too, but have to be distinguished from the admin one somehow.